stepper.py

```diff
@@ -22,20 +23,20 @@ def __init__(self, bot):
         self.steplimit2 = self.steplimit**2
         self.origin_lat = self.bot.position[0]
         self.origin_lon = self.bot.position[1]
+        self.log = logging.getLogger('stepper')
```

Comment:
@berend can you update all `getLogger` calls related to the bot and workers to have simple and meaningful names like this?